recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "spyder-kernels" %}
-{% set version = "2.4.2" %}
-{% set hash = "97586eabda1f714c26035b5b01ed7c2e3f539b0de76bd479af5003f5f19b3eb7" %}
+{% set version = "2.4.3" %}
+{% set hash = "d1a364ab89da716d91671ba9d89ef8f1966068bba0e470797a489653829caaf3" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
Update jupyter-client to `jupyter-client>=7.4.9,<9`, see https://github.com/spyder-ide/spyder-kernels/blob/v2.4.3/setup.py#L47

Reply:
done